ob-pattern-5/templates/obiam/wso2ob-pattern-5-obiam-deployment.yaml

```diff
@@ -0,0 +1,103 @@
+# ----------------------------------------------------------------------------------------
+#
+# Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+#
+# This software is the property of WSO2 LLC. and its suppliers, if any.
+# Dissemination of any information or reproduction of any material contained
+# herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+# You may not alter or remove any copyright or other notice from copies of this content.
+#
+# ----------------------------------------------------------------------------------------
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "ob-pattern-5.resource.prefix" . }}-obiam-deployment
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
```

Comment:
@Chathuratharushka @DivyaPremanantha since IS is scalable shall we set this as an input value that can be passed via the `values.yaml` of the Chart?

Also, let's decide on the number of replicas we are going to go ahead with in each product profile before merging this PR.

Ideally we should support high availability in these production grade Helm Charts.

Reply:
@chirangaalwis Hi we decided to go with 1 replica set from each product. if someone wants to scale it up we are planing provide instructions in documentation how to scale this Kubernetes deployment.

I changed both servers' replica count as the value arguments. https://github.com/wso2/kubernetes-open-banking/pull/76/commits/53c7db9313a36b471cb6fda088d3a4024352bb40